packages/model-viewer/src/features/environment.ts

```diff
@@ -126,6 +126,27 @@ export const EnvironmentMixin = <T extends Constructor<ModelViewerElementBase>>(
         this[$cancelEnvironmentUpdate] = null;
       }
 
+      // Update background color before to get texture utils because renderer singleton depends on background transparency
+      if (skyboxImage == null)
+      {
+        if (backgroundColor === 'transparent') {
+          if (this[$scene].background != null) {
+            this.disconnectedCallback();
+            delete this[$scene].background;
```

Comment:
If this isn't significantly different from assigning `null`, prefer to assign `null` instead

Reply:
ok, that's fixed.